packages/sources/peregrine-fund-admin/tsconfig.test.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Also need to be modify packages/sources/tsconfig.test.json

Reply:
Please provide more suggestions, not sure what requires modification.

Reply:
For example: https://github.com/smartcontractkit/external-adapters-js/pull/3597/files#diff-73224bfa301c6a857d1475a7b853effdd200f9cb7aacfc9b55aa3774628e3e0c

Reply:
Fixed for both and cleaned up.